pkgs/by-name/dr/druid/package.nix

```
@@ -0,0 +1,66 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, extensions ? { }
+, libJars ? [ ]
+, nixosTests
+, mysqlSupport ? true
+}:
+let
+  inherit (lib) concatStringsSep mapAttrsToList optionalString forEach;
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "apache-druid";
+  version = "29.0.0";
+  src = fetchurl {
+    url = "mirror://apache/druid/${finalAttrs.version}/${finalAttrs.pname}-${finalAttrs.version}-bin.tar.gz";
```

```suggestion
    url = "mirror://apache/druid/${finalAttrs.version}/apache-druid-${finalAttrs.version}-bin.tar.gz";
```

`finalAttrs.pname` in src urls is an antipattern